src/py/flwr/client/supernode/app.py

```diff
@@ -67,7 +72,7 @@ def run_supernode() -> None:
         authentication_keys=authentication_keys,
         max_retries=args.max_retries,
         max_wait_time=args.max_wait_time,
-        partition_id=args.partition_id,
+        node_config=parse_config_args(args.node_config),
```

Comment:
We should change the argument names in `parse_config_args` to reflect this use case as well

Reply:
changed input arg of `parse_config_args` from `config_override` to simply `config`.